crates/apub/src/api/read_person.rs

```diff
@@ -69,7 +69,7 @@ impl PerformApub for GetPersonDetails {
       .saved_only(saved_only)
       .local_user(local_user.as_ref())
       .community_id(community_id)
-      .is_mod_or_admin(is_admin)
+      .include_removed(is_admin)
```

Comment:
We can also do `include_deleted` analogously when viewing the own profile. Would be an elegant fix for https://github.com/LemmyNet/lemmy/issues/2624